.github/workflows/track_benches.yml

```diff
@@ -21,80 +21,61 @@ jobs:
+      # https://bencher.dev/docs/explanation/thresholds/#percentage-upper-boundary
+      UPPER_BOUNDARY: 1.0
     steps:
-      - name: Download Benchmark Results
-        uses: actions/github-script@v6
+      - name: Download PR Benchmark Results
+        uses: dawidd6/action-download-artifact@v6
         with:
-          script: |
-            async function downloadArtifact(artifactName) {
-              let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                run_id: context.payload.workflow_run.id,
-              });
-              let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
-                return artifact.name == artifactName
-              })[0];
-              if (!matchArtifact) {
-                core.setFailed(`Failed to find artifact: ${artifactName}`);
-              }
-              let download = await github.rest.actions.downloadArtifact({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                artifact_id: matchArtifact.id,
-                archive_format: 'zip',
-              });
-              let fs = require('fs');
-              fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/${artifactName}.zip`, Buffer.from(download.data));
-            }
-            await downloadArtifact(process.env.PR_BENCHMARK_RESULTS);
-            await downloadArtifact(process.env.BASE_BENCHMARK_RESULTS);
-            await downloadArtifact(process.env.GITHUB_EVENT);
-      - name: Unzip Benchmark Results
-        run: |
-          unzip $PR_BENCHMARK_RESULTS.zip
-          unzip $BASE_BENCHMARK_RESULTS.zip
-          unzip $GITHUB_EVENT.zip
+          name: ${{ env.PR_BENCHMARK_RESULTS }}
+          run_id: ${{ github.event.workflow_run.id }}
+      - name: Download Base Benchmark Results
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          name: ${{ env.BASE_BENCHMARK_RESULTS }}
+          run_id: ${{ github.event.workflow_run.id }}
+      - name: Download GitHub Event
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          name: ${{ env.GITHUB_EVENT }}
+          run_id: ${{ github.event.workflow_run.id }}
       - name: Export GitHub Event Data
         uses: actions/github-script@v6
         with:
           script: |
             let fs = require('fs');
             let githubEvent = JSON.parse(fs.readFileSync("event.json", {encoding: 'utf8'}));
             console.log(githubEvent);
-            core.exportVariable("PR_HEAD", `${githubEvent.pull_request.head.ref}-${githubEvent.pull_request.head.sha.slice(0, 8)}`);
-            core.exportVariable("PR_ID", `${githubEvent.pull_request.head.ref}/${process.env.BENCHER_TESTBED}/${process.env.BENCHER_ADAPTER}`);
-            core.exportVariable("PR_NUMBER", githubEvent.number);
+            core.exportVariable("PR_HEAD", prEvent.pull_request.head.ref);
```

Comment:
The variable `prEvent` is not present here, we can use the `githubEvent` instead.

Reply:
Great catch! Fixed.